druid/src/command.rs

```diff
@@ -59,31 +50,27 @@ pub struct Selector(&'static str);
 ///
 /// [`Command::new`]: #method.new
 /// [`Command::get_object`]: #method.get_object
+/// [`SingleUse`]: struct.SingleUse.html
 /// [`Selector`]: struct.Selector.html
 #[derive(Debug, Clone)]
 pub struct Command {
     /// The command's `Selector`.
     pub selector: Selector,
-    object: Option<Arg>,
+    object: Option<Arc<dyn Any>>,
 }
 
-#[derive(Debug, Clone)]
-enum Arg {
-    Reusable(Arc<dyn Any>),
-    OneShot(Arc<Mutex<Option<Box<dyn Any>>>>),
-}
+/// `SingleUse` is intended for cases where a command payload should only be
```

Comment:
huh I think I lost a comment here?? anyway, to try again:

Doc comments are kind of like git commit messages; you want the first paragraph to be a short general overview that doesn't need a lot of context, and then a blank line, and then a more detailed explanation.

This is because the first paragraph is displayed beside the type name in the module or crate level docs.

For this type, for instance, I would try a first paragraph of something to the effect of,

"A wrapper type for `Command` arguments that should only be used once."

Reply:
Thanks, I should have actually built the docs :sweat_smile: 

I've added the suggested head paragraph, and while I was at it also an example how to make use of `SingleUse`.